src/services/actionValidator/documentActionValidator.tsx

```diff
@@ -23,6 +24,7 @@ const documentActionSchema = yup
       .required()
       .url(),
     key: yup.string(),
+    expiry: yup.number(),
```

Comment:
We are sending the expiry via QR code also? Thinking might not be useful and takes up more space on the QR. every additional character reduce the redundancy on the QR code and the code is less “scannable”. Thinking that the client should just try to fetch the resource and it it gets, it gets.